doc/basic.md

```diff
@@ -187,6 +187,25 @@ ch.close()
 yield csp.take(ch); // csp.CLOSED
 ```
 
+### `offer(ch, value)` ###
+
+Put a value in a channel iff it's possible to do so immediately. Returns `true` if channel received the value, `undefined` otherwise.
+```javascript
+var ch = csp.chan(1);
+csp.offer(ch, 42); // true
+csp.offer(ch, 43); // undefined
+```
+
+### `poll(ch)` ###
+
+Take a value from a channel iff it it's possible to do so immediately. Returns value if succesful, `undefined` otherwise.
```

Comment:
I think it should be a special value not `undefined`, because `undefined` can be put into a channel. If we don't want to distinguish empty channels from closed channels, we can use `CLOSED`, but that is probably confusing.